### PR TITLE
Fix the `Missing parameters` error on the Manager Logs view

### LINKED ISSUE #4089: Missing parameters error in Logs menu
| Wazuh | Elastic | Rev  | Security |
| ----- | ------- | ---- | -------- |
| 4.3   | 7.x     | 4301-1 | Basic, ODFE, Xpack |

| Browser |
| --- |
| Firefox|

### Description

`Missing parameters` error appears when we access to the log menu in the demo environment.

missing_parameters_logs_errors

### PR DESCRIPTION
**Pull request by AlexRuiz7**
## Summary

Closes #4089 

missing-params

## To test

- Go to the Manager logs section.
- No error should appear on start.

### CODE REVIEW

**Review by Mayons — APPROVED** [review]

File: public/controllers/management/components/management/mg-logs/logs.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ export default compose(
 
     async componentDidMount() {
       try {
-        this.height = window.innerHeight - this.offset;
+        this.F = window.innerHeight - this.offset;
```
Comment (yenienserrano) [inline public/controllers/management/components/management/mg-logs/logs.js:84]: is this for something?

Comment (AlexRuiz7) [reply public/controllers/management/components/management/mg-logs/logs.js:84]: Check the actual file, for some reason Github doesn't show the latest version.

Comment (yenienserrano) [reply public/controllers/management/components/management/mg-logs/logs.js:84]: when I clone the branch `this.F` appeared

Comment (AlexRuiz7) [reply public/controllers/management/components/management/mg-logs/logs.js]: Solved. Please review again.

**Review by yenienserrano — APPROVED** [review]
Test: ✔️ 
CR: ✔️

**Pull request merged**